app/facades/foods_facade.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsFacade
+  def self.service
+    @_service ||= FoodsService.new
+  end
+
+  def self.search_by_keyword(word)
+    filtered_results = []
+    
+    service.search_by_keyword(word)[:foods].each do |food|
+      if food[:ingredients].downcase.include?(word)
+        filtered_results << food
+      end
+    end
+    filtered_results.first(10).map do |food|
```

Comment:
Because you added the filtering (which was actually an extension - we caused some confusion there - so nice work!) I think it makes sense to grab the first 10. However, if we could assume that all food hits would come with ingredients that are non-empty, we'd prefer to limit the result set through the API call with a query param (check out the docs to explore how that would work). But for this case, this limiting makes total sense.